lib/tasks/no_mentor_reminders.rake

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+namespace :no_mentor_reminders do
+  desc "Send reminder to all SITs whose school has at least one ECT with no mentor associated"
+  task send: :environment do
+    Schools::WithEctsWithNoMentorQuery.call.each do |school, ects|
+      SchoolMailer.remind_sit_to_set_mentor_to_ects(
+        sit: school.induction_coordinators.first,
+        ect_names: ects.map(&:full_name),
+      ).deliver_later
+    end
+  end
+
+  desc "Print a list of all SITs whose school has at least one ECT with no mentor associated"
```

Comment:
Sure, the plan is to run this task once, get the list printed and then they can decide what subset to send reminders on each bulk. 
So once we get the list we can remove this task because it won't be needed anymore

Reply:
Actually I have printed the list already via the console, so I am removing this task.